tools/devops/automation/templates/agent-pool-selector.yml

```diff
@@ -49,7 +49,7 @@ steps:
       Write-Host "  isTopicBranch: ${isTopicBranch}"
       Write-Host "  isPullRequest: ${isPullRequest}"
 
-      if ($isTopicBranch -or $isPullRequest) {
+      if ($isPullRequest) {
```

Comment:
Looks like the `isTopicBranch` variable isn't used anymore, so it can be removed (and the comments just below this should be updated to reflect the actual behavior as well).